README.textile

```diff
@@ -102,6 +102,10 @@ If you've added and commited everything, check again if all is ok:
 
 This should result in *BUILD SUCCESS*. Push your changes. You're done :)
 
+h2. Tables as gitsubmodules
+The lookup tables are provided through gitsubmodules. To make a @git pull@ also
```

Comment:
Oh, perhaps say "Some lookup tables" since not all of them are provided via submodule.

Reply:
Resoved with [8338ca9](https://github.com/hbz/lobid-resources/pull/1902/commits/8338ca93ec74af0db9c9ebc687ff32aea5b04856).